lib/tapioca/cli.rb

```diff
@@ -335,18 +367,12 @@ def check_shims
       desc: "Override for typed sigils for pulled annotations",
       default: {}
     def annotations
-      if !options[:netrc] && options[:netrc_file]
-        raise Thor::Error, set_color("Options `--no-netrc` and `--netrc-file` can't be used together", :bold, :red)
-      end
-
-      command = Commands::Annotations.new(
-        central_repo_root_uris: options[:sources],
-        auth: options[:auth],
-        netrc_file: netrc_file(options),
-        typed_overrides: options[:typed_overrides],
+      raise Thor::Error, set_color(
```

Comment:
nit: Doesn't 'deprecated' generally mean that a behaviour is still available, just not approved of?